11ty/definitions/misogynoir.md

```diff
@@ -1,5 +1,24 @@
 ---
 title: Misogynoir
 slug: misogynoir
-defined: false
----
\ No newline at end of file
+defined: true
+excerpt: Misogynoir describes the impact of misogyny and anti-Black racism on Black women
+speech: noun
+reading:
+  - text: "They aren’t talking about me…"
+    href: https://www.crunkfeministcollective.com/2010/03/14/they-arent-talking-about-me/
+  - text: "Explanation Of Misogynoir"
+    href: https://www.gradientlair.com/post/84107309247/define-misogynoir-anti-black-misogyny-moya-bailey-coined
+  - text: "On misogynoir: citation, erasure, and plagiarism"
+    href: https://www.gradientlair.com/post/84107309247/define-misogynoir-anti-black-misogyny-moya-bailey-coined
+---
+
+Coined by queer, Black feminist scholar Moya Bailey in 2008, misogynoir is a blend of the words “misogyny” (prejudice and discrimination towards women) and “noir” (a French word meaning Black). Misogynoir describes the impact of misogyny and anti-Black racism on Black women. Though coined by Moya Bailey, Trudy (aka @thetrudz) of Gradient Lair also impacted the spread and usage of the term.
```

Comment:
Typically here we have the "dictionary-style entry" at top followed by the context, as you have. A restructure could look something like this:

```suggestion
the impact of misogyny and anti-Black racism on Black women

## Origins
Coined by queer, Black feminist scholar Moya Bailey in 2008, misogynoir is a blend of the words “misogyny” (prejudice and discrimination towards women) and _noir_ (a French word meaning Black). Though coined by Moya Bailey, Trudy (aka @thetrudz) of Gradient Lair also impacted the spread and usage of the term.
```

Reply:
Gotchaaa. Updating now.